manifests/repo/apt.pp

```diff
@@ -77,7 +77,12 @@
             /i\d86/      => 'i386',
             default      => false,
           }
-          $repo_url  = "http://download.gluster.org/pub/gluster/glusterfs/${release}/LATEST/Debian/${::lsbdistcodename}/apt/"
+          if versioncmp("${release}", '3.12') < 0 { 
```

Comment:
I think you can just use ```$release``` and leave the quotes + braces out.

Reply:
Ack